www/src/components/landingPage/tweets/tweetSlider.tsx

```diff
@@ -1,42 +1,77 @@
-// Import Swiper React components
-import { Swiper, SwiperSlide } from "swiper/react";
-
-// Import Swiper styles
-import "swiper/css";
-import "swiper/css/pagination";
-import "swiper/css/autoplay";
-import "../../../styles/swiper.css";
-import { Autoplay, Pagination } from "swiper";
 import type { Tweet } from "./types";
+import useEmblaCarousel from "embla-carousel-react";
```

Comment:
quick look at the docs suggest this doesn't have to be react?

Reply:
ye the `embla-carousel-react` is just a thin wrapper.
we could also implement the whole component in vanilla js if thats preferred?

Reply:
Yea if possible just make it an Astro component

Reply:
We would loose the ability to this component as an island tho, is that what we want?